### PR TITLE
[YUNIKORN-2000]Adding Chinese translation of Architecture

### PR DESCRIPTION
**Pull request by 0lai0**
### What is this PR for?
The Chinese translation for the document Design of Architecture
in  current development_guide/design/[Architecture](https://yunikorn.apache.org/docs/1.0.0/design/architecture)

### What type of PR is it?
* [ ] - Bug Fix
* [x] - Improvement
* [ ] - Feature
* [x] - Documentation
* [ ] - Hot Fix
* [ ] - Refactoring

### What is the Jira issue?
[YUNIKORN-2000]Adding Chinese translation of Architecture
* [https://issues.apache.org/jira/browse/YUNIKORN-2000](https://issues.apache.org/jira/browse/YUNIKORN-2000)

### Screenshots (if appropriate)
image

### CODE REVIEW

File: i18n/zh-cn/docusaurus-plugin-content-docs/current/design/architecture.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+under the License.
+-->
+
+Apache YuniKorn是一个轻量级、通用的容器编排资源调度器，旨在在大规模、多租户和云原生环境中高效实现各种工作负载的细粒度资源共享。YuniKorn为由无状态批处理工作负载和有状态服务组成的混合工作负载提供了统一的、跨平台的调度体验。
+
+YuniKorn目前支持K8s，并可以部署为自定义的K8s调度器。YuniKorn的架构设计还允许添加不同的Shim层，并适应不同的ResourceManager实现，包括Apache Hadoop YARN或其他系统。
+
+## 架构
+
+以下图表展示了YuniKorn的高级架构。
```
Comment (0yukali0) [inline i18n/zh-cn/docusaurus-plugin-content-docs/current/design/architecture.md:31]: 以下图表展示了YuniKorn的架构。

**Review by 0yukali0 — CHANGES_REQUESTED** [review]

**Review by 0yukali0 — APPROVED** [review]
+1 LGTM

**Review by 0yukali0 — CHANGES_REQUESTED** [review]
I noticed some markdown links are broken.
These warning problems should be resolved.
```
[WARNING] Docs markdown link couldn't be resolved: (../design/architecture.md) in "/yunikorn-site/docs/archived_design/k8shim.md" for version current
[WARNING] Docs markdown link couldn't be resolved: (../archived_design/scheduler_core_design.md) in "/yunikorn-site/i18n/zh-cn/docusaurus-plugin-content-docs/current/design/architecture.md" for version current
[WARNING] Docs markdown link couldn't be resolved: (../archived_design/k8shim.md) in "/yunikorn-site/i18n/zh-cn/docusaurus-plugin-content-docs/current/design/architecture.md" for version current
```

**Comment by wilfred-s** [comment]
No longer relevant after [YUNIKORN-2411](https://issues.apache.org/jira/browse/YUNIKORN-2411)

**Pull request closed without merging**